examples/multiplatform/AppClip/Entitlements.entitlements

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>application-identifier</key>
+	<string>TEAMID.APP_CLIP_BUNDLE_ID</string>
```

Comment:
Having to set this for BwB, but not BwX, continues to stump me. Ideally rules_apple would handle that, but then you would have to know info from your parent app, which is tricky. Maybe even just failing to build if it's not defined...

Reply:
+1 to this. We have a macro that works around this annoying issue.

Reply:
https://github.com/bazelbuild/rules_apple/issues/1570